docs/src/docs/uml/invalidation_strong.txt

```diff
@@ -0,0 +1,68 @@
+@startuml
```

Comment:
If this file gets its extension changed to `.puml` then the intellij plugin picks it up automatically.